package.json

```diff
@@ -36,6 +36,7 @@
     "joi": "^10.6",
     "kafka-node": "^1.6.0",
     "node-schedule": "^1.2.0",
+    "node-uuid": "^1.4.8",
```

Comment:
Just a heads up, recently tried changing this module to use `uuid` because of deprecation warning: https://github.com/scality/S3/pull/1051

Reply:
Thanks for the heads up. Updated.